StreamChatUITestsAppUITests/MockServer/MessageList.swift

```diff
@@ -40,9 +65,15 @@ extension StreamMockServer {
     }
     
     func findMessagesByChannelId(_ channelId: String) -> [[String: Any]] {
-        return messageList.filter {
-            String(describing: $0[MessagePayloadsCodingKeys.cid.rawValue]).contains(":\(channelId)")
+        let cid = MessagePayloadsCodingKeys.cid.rawValue
+        let ephemeralMessages = ephemeralMessageList.filter {
+            String(describing: $0[cid]).contains(":\(channelId)")
+        }
+        var messages = messageList.filter {
+            String(describing: $0[cid]).contains(":\(channelId)")
         }
+        messages += ephemeralMessages
+        return messages
```

Comment:
Do we expect the array of message returned from this func to be sorted by `createdAt`?

Reply:
Do we need to do it on the mock server side? Looks like it works as expected, I thought it's properly sorted by the SDK.